### PR TITLE
TTT-1891 - Added s3 bucket config in order to retrieve credentials fr…

### PR DESCRIPTION
**Pull request by LucianTuduce**
…om aws

#### What tests were added?
None. There are only config files.
#### Outstanding questions?
None.
#### Definition of Done:
- [ ] Is there appropriate test coverage?
- [ ] Does the confluence need to be updated?
- [ ] Does this add new dependencies? If so, have the healthchecks been updated?
- [ ] Any Deployment changes we need to be aware of? New infrastructure? Changes to existing components?
- [ ] Is there appropriate logging included?
- [ ] Does it imply any kind of data migration? If so what are the steps for that?
- [ ] Is the Documentation been updated, if needed?

### CODE REVIEW

File: src/main/resources/application.conf [hunk 1]
```diff
@@ -9,14 +9,28 @@ app {
...
-    baseUrl                : "https://apim.expedia.com/packages/listings?"
-    key                    : "955c4815-347a-44c1-8de0-1fa1d3e768f9"
-    authorization          : "Basic OTU1YzQ4MTUtMzQ3YS00NGMxLThkZTAtMWZhMWQzZTc2OGY5OlNaYXVEMVR2QndDcmRqSnA="
-    accept                 : "application/vnd.exp-package.v3+json"
-    partnerTransactionId   : "user2"
+  package {
+    baseUrl                     : "https://apim.expedia.com"
+    packagesUri                 : "packages/listings?"
+    headers                     {
+      Key                         : ${PACKAGE_KEY}
```
Comment (alin-timu) [inline src/main/resources/application.conf]: Add `?` in order to get env vars. E.g.: ${?PACKAGE_KEY}

File: src/main/resources/application.conf [hunk 2]
```diff
@@ -9,14 +9,28 @@ app {
...
+      Authorization               : ${PACKAGE_AUTHORIZATION}
```
Comment (alin-timu) [inline src/main/resources/application.conf]: Same as above

File: src/main/resources/application.conf [hunk 3]
```diff
@@ -9,14 +9,28 @@ app {
...
+      Accept                      : "application/vnd.exp-package.v3+json"
+      Partner-Transaction-ID      : ${PACKAGE_PARTNERID}
```
Comment (alin-timu) [inline src/main/resources/application.conf]: Same as above

File: infrastructure/dynamic/templates/template.json [hunk 4]
```diff
@@ -0,0 +1,312 @@
...
+                    "EnvVariableName3"            : "PACKAGE_PARTNERID",
+                    "EnvVariableValue3"           : {"Fn::GetAtt":["GetSecrets", "PartnerTransactionID"]},
+                },
+                "TimeoutInMinutes": "15"
+            },
+            "DependsOn": [
+                "ECSServiceTaskRole"
+            ]
+        },
+        "ECSLambda": {
```
Comment (alin-timu) [inline infrastructure/dynamic/templates/template.json]: There is no `template-lambda.json` yet. Remove this or the cloudformation validation will fail.

File: infrastructure/dynamic/templates/template.json [hunk 5]
```diff
@@ -0,0 +1,312 @@
...
+                }]},
+                "Parameters":{
+                    "ApplicationName"           : {"Ref": "ApplicationName"},
+                    "VpcSubNets"                : {"Ref": "VpcSubnets"},
+                    "VpcSecurityGroups"         : {"Ref": "VpcSecurityGroup"}
+                },
+                "TimeoutInMinutes": "30"
+            }
+        },
+        "ECSTriggerUpdate": {
```
Comment (alin-timu) [inline infrastructure/dynamic/templates/template.json]: Same as above, there isn't `template-trigger.json` in the project yet.

File: infrastructure/dynamic/parameters/parameters-preprod.json [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+        "ParameterKey": "VpcSubnets",
+        "ParameterValue": "subnet-a46fcdd2,subnet-bb20dce3,subnet-80823de4"
+    },
+    {
+        "ParameterKey": "VpcSecurityGroup",
+        "ParameterValue": "sg-951180f2"
+    },
+    {
+        "ParameterKey": "BatchHost",
+        "ParameterValue": "platforms-prod.api-platforms.telegraph.co.uk"
```
Comment (alin-timu) [inline infrastructure/dynamic/parameters/parameters-preprod.json]: The host for preprod is `alb.api-platforms-preprod.telegraph.co.uk`

Comment (LucianTuduce) [reply infrastructure/dynamic/parameters/parameters-preprod.json]: Done

File: infrastructure/dynamic/parameters/parameters-preprod.json [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+    },
+    {
+        "ParameterKey": "BatchPath",
+        "ParameterValue": "/travel-hotel-search/batch"
```
[the remark this answers is not in the document]
Comment (LucianTuduce) [reply infrastructure/dynamic/parameters/parameters-preprod.json]: Done

File: infrastructure/dynamic/parameters/parameters-prod.json [hunk 8]
```diff
@@ -0,0 +1,62 @@
...
+        "ParameterKey": "VpcSecurityGroup",
+        "ParameterValue": "sg-951180f2"
+    },
+    {
+        "ParameterKey": "BatchHost",
+        "ParameterValue": "platforms-prod.api-platforms.telegraph.co.uk"
+    },
+    {
+        "ParameterKey": "BatchPath",
+        "ParameterValue": "/travel-hotel-search/batch"
```
[the remark this answers is not in the document]
Comment (LucianTuduce) [reply infrastructure/dynamic/parameters/parameters-prod.json]: Done

File: infrastructure/dynamic/templates/template.json (hunk 5 above)
Comment (LucianTuduce) [reply infrastructure/dynamic/templates/template.json]: Removed

File: infrastructure/static/parameters/parameters-static.json [hunk 9]
```diff
@@ -0,0 +1,14 @@
+[
+    {
+        "ParameterKey"  : "ApplicationName",
+        "ParameterValue": "travel-products"
```
[the remark this answers is not in the document]
Comment (LucianTuduce) [reply infrastructure/static/parameters/parameters-static.json]: Done

File: src/main/resources/application.conf (hunk 1 above)
Comment (LucianTuduce) [reply src/main/resources/application.conf]: Done

File: src/main/resources/application.conf (hunk 2 above)
Comment (LucianTuduce) [reply src/main/resources/application.conf]: Done

File: src/main/resources/application.conf (hunk 3 above)
Comment (LucianTuduce) [reply src/main/resources/application.conf]: Done

File: infrastructure/dynamic/templates/template.json (hunk 4 above)
Comment (LucianTuduce) [reply infrastructure/dynamic/templates/template.json]: Removed

**Pull request merged**